LiskHQ/lisk-sdk
# Fix plugin loading in child process and prevent application from crash if child process crashes - Closes #3142

### What was the problem?

This PR resolves #3142

### How was it solved?

Fixed loading of child process plugins and prevent application from crashing. 

### How was it tested?

1. Run the test app with `node framework/test/test_app/index.js`
2. Check process tree with `htop` or any other utility 
3. You will notice a process named `devnet` and a child process running under as http-api-plugin
4. Kill that child process for http-api-plugin
5. Check if app shows proper log for plugin to exit
6. Check if app still keep running
